grpc/Cargo.toml

```diff
@@ -74,3 +74,5 @@ wasm-bindgen-test.workspace = true
 default = []
 wasm-bindgen = ["celestia-types/wasm-bindgen", "dep:js-sys", "dep:wasm-bindgen", "dep:wasm-bindgen-futures"]
 uniffi = ["dep:uniffi", "dep:async-trait", "celestia-types/uniffi"]
+tls-native-roots = ["tonic/tls-native-roots"]
+tls-webpki-roots = ["tonic/tls-webpki-roots"]
```

Comment:
Issue here is that we cannot enable either of tls-* flags by default, as it'll enable it by default for all platforms - including breaking wasm32.